src/Directory.Build.props

```diff
@@ -5,6 +5,8 @@
     <SkipArcadeNoWarnCS1591>true</SkipArcadeNoWarnCS1591>
     <EnablePackageValidation Condition="'$(EnablePackageValidation)' == ''">true</EnablePackageValidation>
     <PackageValidationBaselineVersion Condition="'$(EnablePackageValidation)' == 'true' and '$(PackageValidationBaselineVersion)' == ''">$(BaselineVersionForPackageValidation)</PackageValidationBaselineVersion>
+    <!-- For Aspire.Hosting.Azure* packages, we need to ingest the new Azure.Provisioning stable packages in order for us to produce stable  bits. -->
+    <SuppressFinalPackageVersion Condition="$(MSBuildProjectName.StartsWith('Aspire.Hosting.Azure'))">true</SuppressFinalPackageVersion>
```

Comment:
This is temporary and will get removed once we ingest the stable version of Azure.Provisioning. cc: @eerhardt @davidfowl